src/Compilers/VisualBasic/Portable/Symbols/Source/QuickAttributeChecker.vb

```diff
@@ -155,6 +155,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                 result = result Or QuickAttributes.Obsolete
             ElseIf Matches(name, inAttribute, AttributeDescription.DeprecatedAttribute) Then
                 result = result Or QuickAttributes.Obsolete
+            ElseIf Matches(name, inAttribute, AttributeDescription.WindowsExperimentalAttribute) Then
+                result = result Or QuickAttributes.Obsolete
```

Comment:
It looks like the `WindowsExperimentalAttribute` case and the `ExperimentalAttribute` case are identical. Consider dropping one.

Reply:
Many of the cases in here are identical. I'd rather stick to the existing structure.